.github/workflows/blogs-indexing.yaml

```diff
@@ -0,0 +1,46 @@
+name: Indexing blogs
+on:
+  push:
+    branches: ['search-blogs-ci-indexing']
+    paths:
+      - 'content/blog/**'
```

Comment:
It will only get triggered changes on `content/blog/**`. I can there is `content/author` too. Should we consider that as well ?